README.md

```diff
@@ -311,6 +312,27 @@ so the above could also be specified in this format:
 generate_hashes = true
 ```
 
+Configuration defaults specific to `pip-compile` and `pip-sync` can be put beneath
+separate sections. For example, to by default perform a dry-run with `pip-compile`:
+
+```toml
+[tool.pip-compile]
+dry-run = true
+```
+
+This does not affect the `pip-sync` command, which also has a `--dry-run` option.
+Note that tool-specific configuration overrides global settings, thus this would
```

Comment:
Let's be more specific than "overrides". When I read this, I thought that the entire global section is discarded. We need to choose wording to highlight that the sections are merged and individual specific settings supersede / take precedence over the global ones.

Reply:
How about `conflicting settings in the global section get overriden by the local ones`?

Reply:
"The local settings take preference over the global ones of the same name, whenever both are declared"?